BWAPI4J/src/main/java/org/openbw/bwapi4j/unit/Building.java

```diff
@@ -1,166 +1,34 @@
-////////////////////////////////////////////////////////////////////////////////
-//
-//    Copyright (C) 2017-2018 OpenBW Team

```

Comment:
Copyright notices should be in all Java and C++ files regardless of their contents.